__tests__/app.test.js

```diff
@@ -37,13 +34,55 @@ describe('/api endpoint', () => {
       })
   })
 })
-describe('error handling', () => {
-  test('404 for missing endpoints', () => {
+describe('/api/topics endpoint', () => {
+  test('GET: 200 should return an array of all topics with slug and description properties to the client', () => {
     return request(app)
-      .get('/api/missing')
-      .expect(404)
+      .get('/api/topics')
+      .expect(200)
       .then(({ body }) => {
-        expect(body.msg).toEqual('/api/missing endpoint not found, get /api for a description of available endpoints')
+        expect(body.topics).toHaveLength(3)
+        body.topics.forEach((topic) => {
+          expect(topic).toHaveProperty('slug')
+          expect(topic).toHaveProperty('description')
+        })
       })
   })
 })
+describe('/api/articles/:article_id endpoint', () => {
+  test("GET: 200 should return the article identified by the specified article_id", () => {
+    const expectedArticle = {
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: "2020-07-09T20:11:00.000Z",
+      votes: 100,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+    }
+    return request(app)
+    .get('/api/articles/1')
+    .expect(200)
+    .then(({body}) => {
+      expect(body.article).toEqual(expectedArticle)
+    })
+  })
```

Comment:
This is a really nice test, but I would recommend having a look at objectContaining/toMatchObject. These will allow your tests to be more flexible, in the event you want to add a new property in the articles table this test would currently fail, but using objectContaining/toMatchObject would still allow this test to pass.